src/search.ts

```diff
@@ -84,7 +85,7 @@ export async function searchIndex(config: Config, effects = defaultEffects): Pro
     )
   );
 
-  indexCache.set(config, {json, freshUntil: +new Date() + reindexingDelay});
+  indexCache.set(pages, {json, freshUntil: Date.now() + reindexDelay});
```

Comment:
The cache is keyed by the `pages` instead of the `config` so that if the set of pages change, the search index is recomputed.